python/kvikio/cufile.py

```diff
@@ -7,6 +7,23 @@
 from ._lib import libkvikio  # type: ignore
 
 
+class PyStreamFuture:
```

Comment:
Important to include the lifetime info both here and in the read/write functions. Something like we have in the C++ API:
```
   * The arguments have the same meaning as in `.read()` but returns a `StreamFuture` object
   * that the caller must keep alive until all data has been read from disk. One way to do this,
   * is by calling `StreamFuture.check_bytes_done()`, which will synchronize the associated stream
   * and return the number of bytes read.
```